csrc/device_lower/analysis/non_divisible_split.cpp

```diff
@@ -212,7 +212,9 @@ NonDivisiblePredicateInfo::NonDivisiblePredicateInfo(Fusion* fusion) {
     }
 
     auto def = tv->definition();
-    if (def == nullptr) {
+    // A block quantization is plumbed down to a runtime function.
```

Comment:
@naoyam Please take a look at this. This needs to be disabled. I am looking into alternate solutions.